api/cluster/pdb.go

```diff
@@ -29,39 +30,50 @@ func NewPodDisruptionBudget(modelService *models.Service, componentType string,
 	labels["serving.kserve.io/inferenceservice"] = modelService.Name
 
 	return &PodDisruptionBudget{
-		Name:                     fmt.Sprintf("%s-%s-%s", modelService.Name, componentType, models.PDBComponentType),
+		Name:                     fmt.Sprintf("%s-%s-%s-%s", modelService.ModelName, modelService.ModelVersion, componentType, models.PDBComponentType),
```

Comment:
to confirm my understanding, modelService.Name was `{modelName}-{modelVersion}-{knativeRevision}`, this essentially strip the knative revision so we can apply the patch for a pdb to be reuse?

Reply:
it's not knativeRevision but meriln's model version revision.
modelService.Name is {modelName}-{versionID}-{revisionPrefix}{revisionID}